code/modules/bitrunning/objects/gimmick_disks/_gimmick_disk.dm

```diff
@@ -0,0 +1,112 @@
+
+/**
+ * Bitrunning tech disks which let you load full loadouts into the vdom on first avatar generation.
+ */
+/obj/item/bitrunning_disk/gimmick
+	desc = "A disk containing source code. It can be used to preload gimmick loadouts into the virtual domain."
+	/// The selected loadout that this grants
+	var/datum/bitrunning_gimmick/granted_loadout
+	/// The list of loadouts that this can grant
+	var/list/datum/bitrunning_gimmick/selectable_loadouts
+
+/obj/item/bitrunning_disk/gimmick/load_onto_avatar(mob/living/carbon/human/neo, mob/living/carbon/human/avatar, external_load_flags)
+	if(isnull(granted_loadout))
+		return LOAD_FAILED
+	return granted_loadout.grant_loadout(neo, avatar, external_load_flags)
+
+/obj/item/bitrunning_disk/gimmick/attack_self(mob/user, modifiers)
+	. = ..()
+
+	if(choice_made)
```

Comment:
why not check for granted_loadout? also is that the right name should it be granting loadout or something along those lines?

Reply:
hrmnm, it was _initially_ in parallel with the other disks, but given this one's an actual instance instead of just a type I feel that makes more sense yeah

I also thought about granting vs granted loadout, but granting loadout sounded kinda off to me personally?
Granted as in to be granted felt better to me

Reply:
vibes